.github/workflows/documentation.yml

```diff
@@ -0,0 +1,58 @@
+name: Documentation
```

Comment:
I separated out the docs task from the package workflow so I didn't accidently run the real deploy while testing. It seemed nice to keep it separate.